ledger-core/v2/virtual/execute/execute.go

```diff
@@ -49,6 +53,8 @@ type SMExecute struct {
 	executionNewState *executionupdate.ContractExecutionStateUpdate
 	executionID       uuid.UUID
 	executionError    error
+	outgoingResult    interface{}
```

Comment:
I guess it should be []byte, not interface. All the results are bytes

Reply:
fixed everywhere